internal/commands/scan.go

```diff
@@ -46,6 +46,10 @@ func NewScanCommand() *cobra.Command {
 		accessTokenFlagName, accessTokenShortFlag, "",
 		"the access token to use for the repository")
 
+	scanCommand.PersistentFlags().StringVarP(&scm,
+		scmFlagName, scmShortFlag, "github",
+		"the SCM to use for the repository")
```

Comment:
what do you think about: "the self hosted scm endpoint"

Reply:
In this case it is not just about self-hosted but selection of the type of SCM. The RepositoryURL still provides the host for the endpoint supporting SaaS/dedicated/self-hosted

Reply:
what about "the self hosted SCM vendor"?